### DISCUSSION TITLE
glpi-agent-deployment.vbs not accepting spacing in SetupOptions

### DISCUSSION THREAD

**Discussion by kintaro1981** [glpi-project/glpi-agent#405] [Q&A] [answered]
### Bug reporting acknowledgment

Yes, I read it

### Professional support

None

### Describe the bug

I'm tring to use vbs script with  LOCAL='C:\Program Files\GLPI-Agent' inSetupOptions 
But the VBS script doesn't works.

I found out that the problem is the space in "Program Files".

### To reproduce

Set SetupOptions like this:

`SetupOptions = "/quiet RUNNOW=1 DEBUG=1 ADD_FIREWALL_EXCEPTION=1 LOCAL='C:\Program Files\GLPI-Agent' SERVER=http://myip_or_domain/plugins/glpiinventory"`

and run the VBS script

### Expected behavior

Install GLPI-Agent or riconfigure it with LOCAL as C:\Program Files\GLPI-Agent

### Operating system

Windows

### GLPI Agent version

1.4

### GLPI version

10.0.6

### GLPIInventory plugin or FusionInventory for GLPI plugin version

Not applicable

**Comment by g-bougard** [comment]
Hi @kintaro1981 

be sure to not run the VBS script from a PowerShell console/environment.

**Reply by kintaro1981** [reply]
I'm running it from cmd.exe like:

`cscript glpi-agent-deployment.vbs`

This is the the file (txt because I can't upload vbs files):
[glpi-agent-deployment.txt](https://github.com/glpi-project/glpi-agent/files/11578361/glpi-agent-deployment.txt)

**Reply by g-bougard** [reply, marked as answer]
It seems me to run as expected if you use the following syntax:
`LOCAL=""C:\Program Files\GLPI-Agent""`

So try to double the double-quotes.

**Reply by vboucard** [reply]
I'll try this ! I used the 8dot3 notation (PROGRA~1, as shown with dir /x)
